### PR TITLE
[ARM] az deployment: Fix the parse bug for comment under the last argument (#11995)

### PR DESCRIPTION
**Pull request by zhoxing-ms**
Issue: #11995
When commenting at the bottom of all elements in a JSON object, jsmin has a bug that will wrap lines.
It will affect the subsequent multi-line processing logic, so deal with this situation in advance.

**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S167.

File: src/azure-cli/azure/cli/command_modules/resource/tests/latest/simple_deploy.json [hunk 1]
```diff
@@ -33,8 +33,12 @@
...
   "outputs": {
     "NewNSG": {
-      "type": "object",
-      "value": "[reference(parameters('name'))]"
+      // comment
+      /** comment **/
+      "type": "object", /** comment **/
+      "value": "[reference(parameters('name'))]" // comment
+      // comment
+      /** comment **/
```
Comment (qianwens) [inline src/azure-cli/azure/cli/command_modules/resource/tests/latest/simple_deploy.json]: how about /** comment line1
comment line2
comment line3
**/

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/resource/tests/latest/simple_deploy.json]: Well, this is handled by jsmin(), let me supplement the test of this case~

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/resource/tests/latest/simple_deploy.json]: Updated

**Pull request merged**